### PR TITLE
[CAL-1493] Tasks Schedule

### PR DESCRIPTION
**Pull request by r1dotl**
We would like to introduce tasks.schedule on the Tasks API in order to be able to create an event linked to a task (schedule task).

### CODE REVIEW

File: src/08-tasks/tasks.apib [hunk 1]
```diff
@@ -163,3 +163,22 @@ Delete a task.
         + id: `5f0afd8a-8a40-48a4-bbe6-7d0e9c61bb6d` (string, required)
 
 + Response 204
+
+
```
Comment (WouterSioen) [inline src/08-tasks/tasks.apib]:
```suggestion
```

File: src/changes-backwards-compatible.apib [hunk 2]
```diff
@@ -6,6 +6,10 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- We added `tasks.schedule` endpoint.
```
Comment (WouterSioen) [inline src/changes-backwards-compatible.apib]:
```suggestion
- We added a `tasks.schedule` endpoint.
```

File: src/changes-backwards-compatible.apib [hunk 3]
```diff
@@ -6,6 +6,10 @@ We list all backwards-compatible additions here. These are currently available i
...
+
+- `task_id` filter has been added to `/events.list`.
```
Comment (WouterSioen) [inline src/changes-backwards-compatible.apib]:
```suggestion
- A `task_id` filter has been added to `/events.list`.
```

**Review by WouterSioen — DISMISSED** [review]

File: apiary.apib [hunk 4]
```diff
@@ -371,6 +371,10 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- We added `tasks.schedule` endpoint.
+
+- `task_id` filter has been added to `/events.list`.
```
Comment (jenssegers) [inline apiary.apib:376]: This is part of https://github.com/teamleadercrm/api/pull/432 right?

File: apiary.apib [hunk 5]
```diff
@@ -3444,6 +3449,25 @@ Delete a task.
 
 + Response 204
 
+
+### tasks.schedule [POST /tasks.schedule]
+
+Schedule a task.
```
Comment (jenssegers) [inline apiary.apib:3455]:
```suggestion
Schedule a task in your calendar.
```

Comment (psrpinto) [reply apiary.apib:3455]: PP: I think adding "in your calendar" is confusing, I don't think we have the concept of a calendar today in the API.

File: apiary.apib (hunk 4 above)
Comment (psrpinto) [reply apiary.apib:376]: B: Indeed. Please let's make it so PRs only contain relevant changes.

**Review by psrpinto — CHANGES_REQUESTED** [review]
I'd like to see irrelevant changes removed from the PR

**Comment by r1dotl** [comment]
Closing this one and opening a clean one. For some reason things that wasnt from this PR were commited here.

**Pull request closed without merging**